src/rest-server/src/routes/v2/virtual-cluster.js

```diff
@@ -28,10 +28,6 @@ router.route('/')
   /** GET /api/v2/virtual-clusters - Return cluster virtual cluster info */
   .get(controller.list);
 
-router.route('/nodeResource')
```

Comment:
also changed api /api/v2/virtual-clusters/nodeResource to /api/v2/virtual-clusters?nodes,
in case there's a vc called nodeResource